exercises/secret-handshake/instructions.md

```diff
@@ -0,0 +1,47 @@
+# Instructions
+
+Your task is to convert a number between 1 and 31 to a sequence of actions in the secret handshake.
+
+The sequence of actions is chosen by looking at the rightmost five digits of the number once it's been converted to binary.
+Start at the right-most digit and move left.
```

Comment:
It's *possible* that someone could complain that due to the word origin of the word "digit", "digit" should only be used for base ten, but I think this usage is acceptable without needing to worry about that.

Reply:
Would `rightmost five bits` be better?

Reply:
I don't much like _bit_ because we haven't defined it, and I think we might need to define it if we use it.

Digit is by definition a numerical digit `0` to `9`. While the word comes from the idea that we have 10 fingers, `0` and `1` are both numerical digits, even when used to represent binary.

I _think_ we're good here.

@petertseng would you agree with this?

Does anyone else have any concerns about this (before we merge and sync this to all the tracks)?